src/MainContent/Product.js

```diff
@@ -24,6 +24,7 @@ function Product(props) {
       <div className="product-container" onClick={() => setpopupAcive(true)}>
         <img className="product-image" src={props.imgSrc}></img>
         <div className="product-title">{props.title}</div>
+        <div className="product-price">{props.price} €</div>
```

Comment:
palik čia tiesiog diva be className, nes kokios klases nėr css.(arba as nerandu) Ir prie to pačio išcentruok nuotrauką, nes  kur title ilgas, nuotrauka pasislinkus desinėn